src/main/java/com/swp/config/SecurityConfig.java

```diff
@@ -24,6 +26,11 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
 	private final JwtProvider jwtProvider;
 	private final JwtAuthenticationEntryPoint jwtAuthenticationEntryPoint;
 
+	@Override
+	public void configure(WebSecurity web) throws Exception {
+		web.ignoring().antMatchers("/login/renew");
+	}
```

Comment:
조금 더 상세히 ignore할 path를 지정하는 아래 방식은 어떨까요??
```suggestion
	@Override
	public void configure(WebSecurity web) throws Exception {
		web.ignoring().mvcMatchers(HttpMethod.POST, "/login/renew");
	}
```